### PR TITLE
Catch file syntax error of encrypted data

### PR DESCRIPTION
**Pull request by etchone**
## Proposed Changes

- Fixes alerting "Incorrect password" when parsing content with syntax error

@MikeCoder

### CODE REVIEW

**Comment by etchone** [comment]
example .md file

```markdown
---
title: Example
---

<script lang="javascript">
    function f() {
    //error here
</script>
```

File: lib/blog-encrypt.js [hunk 1]
```diff
@@ -30,7 +30,24 @@ function decryptAES () {
...
+      } catch(e) {
+        
+        var errorInfo = "<p>"
+                      +   "You can see this mainly because there are some error "
+                      +   "with the file encrypted."
+                      + "</p>"
+                      + "<p>"
+                      +   "Please check the original file."
+                      + "</p>";
+        console.error(errorInfo);
```
Comment (xiazeyu) [inline lib/blog-encrypt.js:46]: I think it's better to `console.error(e)`

**Comment by D0n9X1n** [comment]
Nice job.

File: lib/blog-encrypt.js [hunk 2]
```diff
@@ -30,7 +30,24 @@ function decryptAES () {
...
       // Use jquery to load some js code
-      $('#encrypt-blog').html(content);
+      try {
+        
+        $('#encrypt-blog').html(content);
+
+      } catch(e) {
+        
+        var errorInfo = "<p>"
+                      +   "You can see this mainly because there are some error "
```
Comment (xiazeyu) [inline lib/blog-encrypt.js:40]: Some errors occurred, check the original file please.
Detailed exception are shown in console.

**Review by xiazeyu — COMMENTED** [review]
I think these words are better to change.
Function LGTM.

File: lib/blog-encrypt.js (hunk 1 above)
Comment (D0n9X1n) [reply lib/blog-encrypt.js:46]: Agree.

File: lib/blog-encrypt.js (hunk 2 above)
Comment (xiazeyu) [reply lib/blog-encrypt.js:40]: Updated in https://github.com/MikeCoder/hexo-blog-encrypt/commit/862c24bcaf8b277c8b6956b23993a6533a48b52a

File: lib/blog-encrypt.js (hunk 1 above)
Comment (xiazeyu) [reply lib/blog-encrypt.js:46]: Updated in https://github.com/MikeCoder/hexo-blog-encrypt/commit/862c24bcaf8b277c8b6956b23993a6533a48b52a

**Pull request merged**